Cargo.toml

```diff
@@ -14,6 +14,7 @@ categories = ["web-programming::http-server"]
 
 [dependencies]
 actix-web = { version = "4.0.0-beta.9", default-features = false }
+http = "^0.2"
```

Comment:
I don't know about the latest changes in the actix beta, but I think the re-export for `http` is still there so using `actix_web::http` would be better than introducing new deps.

Reply:
Good catch, didn't know that was there!